locales/en-US/app.ftl

```diff
@@ -109,6 +109,52 @@ CallNodeContextMenu--copy-function-name = Copy function name
 CallNodeContextMenu--copy-script-url = Copy script URL
 CallNodeContextMenu--copy-stack = Copy stack
 
+## CallTree
+## This is the component for Call Tree panel.
+
+CallTree--tracing-ms-total = Running Time (ms)
+    .title =
+        The "total" running time includes a summary of all the time where this
+        function was observed to be on the stack. This includes the time where
+        the function was actually running, and the time spent in the callers from
+        this function.
+CallTree--tracing-ms-self = Self (ms)
+    .title =
+        The "self" time only includes the time where the function was
+        the leaf-most one on the stack. If this function called into other functions,
```

Comment:
Is _leaf-most_ equivalent to _doesn't have any children_? If so, maybe that's a little clearer?

Reply:
I think "leaf" is like an established term in computer science for tree data structure. But maybe we should not make any assumptions for our users. I think we should have some explanations for the "Call Tree" in our user docs though.
@julienw what do you think?

Reply:
Yeah, "leaf" is part of the "tree" metaphor, but ultimately here I believe this is more about the stack than about the tree (and TBH when I started working on the project I couldn't understand this term of "leaf").

For each point of data, we have a stack: A calls B which calls C which calls E, that we can write `A -> B -> C -> D`. The leaf-most function here is `D`, this is where the "real" work is done.

For stacks we could use something more like "bottommost". Indeed sometimes what _we_ call `inverted tree` is called `bottom-up`, and what we call the `uninverted tree`  is called `top-down`.

In this specific case, we could say `where the function was the bottom of the stack` for example? Or `where the function was the end of the stack`? (because stacks are sometimes written reversed, maybe the term "end" is more understandable than "bottom"?)

What do you all think?

Reply:
Bottom/end of the stack sounds clearer to me.

Reply:
Ok then, I reworded it to "end of the stack". Thanks!